CHashSet/src/main/java/org/example/CHashSet.java

```diff
@@ -0,0 +1,120 @@
+package org.example;
+
+import java.util.Arrays;
+import java.util.Iterator;
+import java.util.NoSuchElementException;
+
+public class CHashSet<T> implements Iterable<T> {
+    private static final int INITIAL_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+
+    private Object[] table;
+    private int size;
+    private int threshold;
+
+    public CHashSet() {
+        table = new Object[INITIAL_CAPACITY];
+        threshold = (int) (INITIAL_CAPACITY * LOAD_FACTOR);
+    }
+
+    private int getIndex(T element) {
+        return Math.abs(element.hashCode()) % table.length;
+    }
+
+    private void resize() {
```

Comment:
keep please the order of methods inside the class 
private methods should be after public and and better to write it just after the method in which they are called